recipes/gtest/all/conanfile.py

```diff
@@ -58,7 +54,7 @@ def _configure_cmake(self):
         return cmake
 
     def build(self):
-        tools.patch(base_path=self._source_subfolder, patch_file="gtest.patch")
+        tools.patch(base_path=self._source_subfolder, patch_file="gtest-" + str(self.version) + ".patch")
```

Comment:
prefer to use the conandata.yml form: https://github.com/conan-io/conan-center-index/wiki/Error-Knowledge-Base#kb-h030-conandatayml-format